composer.json

```diff
@@ -103,7 +103,8 @@
         "npm-asset/jquery.easing": "^1.4",
         "npm-asset/slick-carousel": "^1.8",
         "oomphinc/composer-installers-extender": "^1.1 || ~2.0.0",
-        "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2"
+        "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
+        "ymcatwincities/openy_activity_finder": "^3.0"
```

Comment:
some day we'll have "^3.0 || ^4.0" and then this line probably should go ))))

Reply:
Right you are ;)